### PR TITLE
Configure database

### LINKED ISSUE #26: Configure database

### PR DESCRIPTION
**Pull request by veca22**

### CODE REVIEW

**Review by balsa-sarenac — APPROVED** [review]

File: sbag/generators/java/templates/__project__/src/main/resources/application.properties.jinja [hunk 1]
```diff
@@ -0,0 +1,8 @@
+spring.datasource.driverClassName=org.postgresql.Driver
+spring.datasource.url= jdbc:postgresql://localhost:5432/test_db
+spring.datasource.initialization-mode=always
+spring.datasource.platform=postgres
+spring.datasource.username=postgres
+spring.datasource.password=ftn
```
Comment (vtanja) [inline sbag/generators/java/templates/__project__/src/main/resources/application.properties.jinja:6]: Should we set password to be enviroment variable?

Comment (vtanja) [inline sbag/generators/java/templates/__project__/src/main/resources/application.properties.jinja]: Should we set password to be enviroment variable?

**Review by vtanja — APPROVED** [review]

File: sbag/generators/java/templates/__project__/src/main/resources/application.properties.jinja (hunk 1 above)
Comment (balsa-sarenac) [reply sbag/generators/java/templates/__project__/src/main/resources/application.properties.jinja]: I agree, both password and username, maybe even database name

**Review by balsa-sarenac — CHANGES_REQUESTED** [review]
Change hardcoded db username, password and name to environment variables

**Review by nikolina-sarenac — APPROVED** [review]

**Review by balsa-sarenac — APPROVED** [review]

**Pull request merged**